File: references/admin_templates/template_1/CakePHP/CakePHP/src/Controller/ChartController.php
<?php
declare(strict_types=1);

namespace App\Controller;

/**
 * Chart Controller
 *
 */
class ChartController extends AppController
{
    public function columnChart()
    {
        $this->set('title', 'Column Chart');
        $this->set('subTitle', 'Components / Column Chart');
    }
    
    public function lineChart()
    {
        $this->set('title', 'Line Chart');
        $this->set('subTitle', 'Components / Line Chart');
    }
        
    public function pieChart()
    {
        $this->set('title', 'Pie Chart');
        $this->set('subTitle', 'Components / Pie Chart');
    }
}

File: references/admin_templates/template_1/MVC/MVC/Controllers/ChartController.cs
using Microsoft.AspNetCore.Mvc;

namespace WowDash.Controllers
{
    public class ChartController : Controller
    {
        public IActionResult ColumnChart()
        {
            return View();
        }
        public IActionResult LineChart()
        {
            return View();
        }
        public IActionResult PieChart()
        {
            return View();
        }
    }
}
